react/game/src/index.css

```diff
@@ -0,0 +1,50 @@
+body {
```

Comment:
U must use `scss` files, as you learned in the previous training. Place each style in the corresponding component, as in the card says, and global styles in `scss` folder as you learned.